prysm/helm/templates/_unique_volumes_helper.tpl

```diff
@@ -0,0 +1,15 @@
+{{- define "unique.volumes" }}
+{{- $volumeDict1 := dict .dataVolumePath "validator" }} 
+{{- $volumeDict2 := dict .walletVolumePath "wallet" }}
+{{- $uniqueVolumePaths := keys $volumeDict1 $volumeDict2 | uniq }}
+{{- $finalDict := dict }}
+{{- if lt (len $uniqueVolumePaths) 2 }}
+  {{- range $path := $uniqueVolumePaths}}
+    {{- $tempValue := pluck $path $volumeDict1 $volumeDict2 | join "-" | printf "prysm-%s-stroage" }}
+    {{- $finalDict := set $finalDict $path $tempValue }}
```

Comment:
Do `{{- $finalDict := set $finalDict $path $tempValue }}` and `{{- $_ := set $finalDict $path $tempValue }}` have the same effect?

Reply:
yes it is. not sure which one is the better practice although I feel the current one is more semantic than trapping it with `$_`